allauth/account/views.py

```diff
@@ -654,6 +654,12 @@ def logout(self):
                                   messages.SUCCESS,
                                   'account/messages/logged_out.txt')
         auth_logout(self.request)
+        signal_kwargs = {}
+        signals.user_logged_out.send(sender=self.request.user.__class__,
+                                    request=self.request,
+                                    user=self.request.user,
```

Comment:
There is not really a decision to take, we need to make sure that the user that was logged out is available as a signal parameter.. when that is taken care of this can be merged.